test/e2e/network/networking.go

```diff
@@ -133,7 +133,7 @@ var _ = SIGDescribe("Networking", func() {
 			// TODO: test proxy links here
 		}
 		if !framework.ProviderIs("gke", "skeleton") {
-			tests = append(tests, struct{ path string }{path: "/logs"})
+			tests = append(tests, struct{ path string }{path: "/logs/"})
```

Comment:
this is unexpected

Reply:
see also https://github.com/kubernetes/kubernetes/pull/93144#issuecomment-659920421